tests/miral/test_window_manager_tools.cpp

```diff
@@ -255,7 +255,7 @@ auto mt::TestWindowManagerTools::create_mock_display_configuration(std::vector<m
 {
     auto const display_config = std::make_shared<const mir::test::doubles::MockDisplayConfiguration>();
     EXPECT_CALL(*display_config, for_each_output(testing::_))
-        .WillOnce(testing::Invoke([outputs](std::function<void(mir::graphics::DisplayConfigurationOutput const&)> func){
+        .WillRepeatedly(testing::Invoke([outputs](std::function<void(mir::graphics::DisplayConfigurationOutput const&)> func){
```

Comment:
If we "Don't test that display_config->for_each_output() is called exactly once" then we should probably have `.Times(AtLeast(1))` instead of an implied `.Times(1)`.

Reply:
As is with `WillRepeatedly`, it can be called any number of times (including zero). I believe that's what we want, because just getting a config doesn't imply if/how many times a test is going to use it.

Reply:
But the second and subsequent calls will needlessly spam the logs with "unexpected call".

Reply:
I created a `FakeDisplayConfiguration` as we discussed